src/kafka/account-generator/AccountGenerator.ts

```diff
@@ -26,16 +26,12 @@ export class AccountGenerator {
    */
   constructor(brokers: string[]) {
     this.producer = new Producer(brokers, {
-      logLevel: logLevel.NOTHING,
       clientId: 'wallet-generator-sdk-producer'
     })
 
-    this.consumer = new Consumer(
-      brokers,
-      [DEFAULT_TOPIC_ACCOUNT_GENERATED],
-      { logLevel: logLevel.NOTHING },
-      { groupId: 'wallet-generator-sdk-consumer' }
-    )
+    this.consumer = new Consumer(brokers, [DEFAULT_TOPIC_ACCOUNT_GENERATED], undefined, {
```

Comment:
Diging into the code I see `undefined` corresponds to `kafkaConfig`, which is used like follow

```
super({ clientId: 'orchestrate-sdk-consumer', ...kafkaConfig, brokers })
```

I think it would be better to pass an empty object `{}`, instead of undefined which, I believe, cannot be destructured